README.md

```diff
@@ -39,7 +39,79 @@ In your application.js:
 ```
 ## Usage
 
-We're not sure yet.
+Given a simple application with Tasks belonging to Projects, the first thing to do is allow your Project model to accepts_nested_attributes_for.
+
+**Models**
+
+```ruby
+class Project < ApplicationRecord
+  has_many :tasks, inverse_of: :project
+  accepts_nested_attributes_for :tasks, reject_if: :all_blank, allow_destroy: true
+end
+
+class Task < ApplicationRecord
+  belongs_to :project
+end
+```
+
+Because Rails 5 and above belongs_to associations are required by default, we must specify the link because the child object(tasks) cannot be saved until the parent (a project) is saved.
+
+Moving on, we'll need to add the parameters to the controller.
+
+**Controller**
+
+```ruby
+# ProjectsController
```

Comment:
`controllers/projects_controller.rb`

Reply:
why? this could be nested if the project is big enough, i just need to provide enough context to ensure they know its a controller